build32.sh

```diff
@@ -9,6 +9,8 @@ git reset --hard
 git pull origin main || true
 cd ../../..
 
+make clean
```

Comment:
Why clean here?

Reply:
Because otherwise, running another build fails. I'm guessing that's because we're messing directly with a build artifact and the next builds pick that up.

Reply:
Yes, the 32 and 64 builds mess with each other.

I also have a clean in the code that builds the release.